MODEL_ZOO.md

```diff
@@ -1,13 +1,21 @@
 # Model Zoo and Baselines
 
 ### Introduction
-We trained ResNet-50 baseline on Cityscapes with 4 1080TI GPUs, with a batchsize of 4.  
+We trained ResNet-50 / HRNet-W48 baseline on Cityscapes with 4 1080TI GPUs / 8 V100 GPUs, with a batchsize of 4 / 8 separately.
 We use the [TorchVision](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py)
 ResNet implementation, which is not exactly the same as the 
 [TensorFlow](https://github.com/tensorflow/models/blob/master/research/deeplab/core/resnet_v1_beta.py) 
 ResNet implementation.
+We use the [HRNet-Semantic-Segmentation](https://github.com/HRNet/HRNet-Semantic-Segmentation) HRNet implementation.
+
+### Cityscapes baselines (Under Progress)
+
+All the results are under progress and we are making efforts to finetune various hyperparameters to achieve better performance. 
+
+We use Pytorch1.5 in all of our experiments. The docker is [rainbowsecret/pytorch1.5:latest](https://hub.docker.com/repository/docker/rainbowsecret/pytorch1.5).
```

Comment:
Maybe use a separate table for ResNet and HRNet backbone? You can associate this with the HRNet table, and we can add result for HRNet18 and 32 in the future.

Reply:
fixed